iml-agent/src/cli.rs

```diff
@@ -545,6 +545,10 @@ async fn main() -> Result<(), Box<dyn std::error::Error>> {
                 target_fs,
                 files,
             } => {
+                if action == action_filesync::ActionType::Pull {
+                    tracing::error!("Pull is not available as an option");
```

Comment:
Is pull just not supported in the CLI, or not at all?

Reply:
Not in V1, but there's a bunch of code in the back end (action_filesync.rs) that's already written, so I'd prefer to do some minor changes to disable rather than write and replace.